ansible/roles/test/files/ptftests/py3/advanced-reboot.py

```diff
@@ -732,6 +732,11 @@ def setUp(self):
         self.log("Enabling arp_responder")
         self.cmd(["supervisorctl", "restart", "arp_responder"])
 
+        # Give arp_responder 15 seconds to start up, because with the libpcap backend, scapy will first get information
+        # about all of the interfaces on the system (which takes a bit of time) and then proceeds.
+        self.log("Waiting 15 seconds for ARP responder to complete initialization")
```

Comment:
If this change is really needed for advanced-reboot then it should also be added to other test cases that use or depend on arp_responder.

Reply:
`which takes a bit of time` - this can potentially cause regressions on other testcases if they do not adapt accordingly too.

Reply:
Based on a quick test, when we migrate to py3-only PTF, there shouldn't be any delays in ARP responder to complete initialization.